### PR TITLE
Add gpu pricing

### PR DESCRIPTION
**Pull request by andie787**
### Summary of changes

- add GPU pricing
- change title of "Apps V2 and Machines" to "Fly Machines"
- remove V1 compute pricing
- update link to new billing page

### Related Fly.io community and GitHub links
https://community.fly.io/t/fly-gpus-are-here/16110

### Notes
n/a

### CODE REVIEW

File: about/pricing.html.markerb [hunk 1]
```diff
@@ -43,17 +43,29 @@ Additional resources are billed at the usage-based pricing detailed below.
...
 
 Pricing for [Fly Machines](/docs/machines/) VMs, which are also the virtual-machine building blocks of the [Fly Apps V2 platform](/docs/reference/apps/#apps-v2) and [Fly Postgres](/docs/postgres/) apps.
 
 <%= partial("shared/cpu_mem_machines_pricing") %>
 
-### Apps V1
+### GPUs and Fly Machines
 
-Pricing for virtual machines running in [Fly Apps V1](/docs/reference/apps/#legacy-apps-fly-apps-v1) (Nomad-orchestrated) apps.
+Pricing for Fly Machines attached to GPUs. GPU access requires a [Launch, Scale, or Enterprise plan](https://fly.io/plans).
```
Comment (catflydotio) [inline about/pricing.html.markerb:54]: This is just the pricing for the GPU part of a Fly Machine attached to a GPU, isn't it?

Comment (vyedin) [reply about/pricing.html.markerb:54]: Yes.

Comment (catflydotio) [inline about/pricing.html.markerb:54]:
```suggestion
Pricing for a GPU-enabled Fly Machine is the price of a standard Fly Machine (see above) plus the price of the attached GPU. GPU access requires a [Launch, Scale, or Enterprise plan](https://fly.io/plans).
```

Meh?

Comment (andie787) [reply about/pricing.html.markerb:54]: @vyedin Does this work? Any other changes needed?

**Pull request merged**